src/k8s/cmd/k8s/k8s_join_cluster.go

```diff
@@ -63,6 +82,8 @@ var (
 func init() {
 	joinNodeCmd.Flags().StringVar(&joinNodeCmdOpts.name, "name", "", "The name of the joining node. defaults to hostname")
 	joinNodeCmd.Flags().StringVar(&joinNodeCmdOpts.address, "address", "", "The address (IP:Port) on which the nodes REST API should be available")
+	joinNodeCmd.Flags().BoolVar(&joinNodeCmdOpts.force, "force", false, "If set, a currently running cluster will be purged before joining.")
```

Comment:
no, let's not do this. there's lots of things that can go wrong if we allow this (e.g. existing runtime configuration being only one of them).

Reply:
discussed offline - this is indeed a bad idea and I remove it.